tests/unit/cluster/failover2.tcl

```diff
@@ -0,0 +1,62 @@
+# Check the basic monitoring and failover capabilities.
+
+start_cluster 3 6 {tags {external:skip cluster} overrides {cluster-ping-interval 1000 cluster-node-timeout 15000}} {
+
+test "Cluster is up" {
```

Comment:
Let's indent this and all other blocks here. We tried to preserve a bunch of history by not indenting in the past, but for net new tests there is no reason to just use the normal style.